helm/alfresco-content-services/values.yaml

```diff
@@ -890,3 +896,16 @@ global:
 #   region:
 #   s3Bucket:
 #   comprehendRoleARN:
+
+  # -- Shared connections details for Elasticsearch/Opensearch, required when alfresco-elasticsearch-connector.enabled is true
+  elasticsearch:
+    # -- The host where service is available. The provided default is for when elasticsearch.enabled is true
+    host: elasticsearch-master
+    # -- The port where service is available
+    port: 9200
+    # -- Valid values are http or https
+    protocol: http
+    # -- The username required to access the service, if any
+    user: null
+    # -- The password required to access the service, if any
+    password: null
```

Comment:
same here. I would have only null values (so you can document values) and use a template that basically outputs:
`es.*` if exists otherwise `global.es.*` otherwise default values as they are computed in existing tpl